android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -8,17 +8,26 @@ import android.annotation.SuppressLint
 import android.content.Context
 import android.graphics.Bitmap
 import android.os.Parcelable
+import androidx.annotation.VisibleForTesting
 import androidx.core.net.toUri
 import kotlinx.parcelize.Parcelize
+import mozilla.components.concept.engine.webextension.Metadata
 import mozilla.components.concept.engine.webextension.WebExtension
+import mozilla.components.support.base.log.logger.Logger
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.TimeZone
+
+val logger = Logger("Addon")
 
 /**
  * Represents an add-on based on the AMO store:
  * https://addons.mozilla.org/en-US/firefox/
  *
  * @property id The unique ID of this add-on.
  * @property author Information about the add-on author.
- * @property categories List of categories the add-on belongs to.
+ * @property downloadId The unique ID of the latest version of the add-on (xpi) file.
```

Comment:
This was removed previously. No need to add it back.

Reply:
Updated!